homeassistant/components/olarm/strings.json

```
@@ -0,0 +1,39 @@
+{
+  "config": {
+    "step": {
+      "device": {
+        "title": "Select Device",
+        "description": "Choose the Olarm device you would like to integrate. If your device is not listed, please ensure you have enabled public API access in your Olarm account for that device.",
+        "data": {
+          "select_device": "Device",
+          "load_zones_bypass": "Would you like to also load entities for zone bypassing?"
```

This should sound like a label, not a dialog question:

```suggestion
          "load_zones_bypass": "Load entities for zone bypassing"
```

You should use the currently empty `data_description` for this field to explain why this is optional.